verticaltabs.js

```diff
@@ -706,7 +706,13 @@ VerticalTabs.prototype = {
     };
     window.addEventListener('aftercustomization', afterListener);
 
-    window.addEventListener('resize', this.resizeTabs.bind(this), false);
+    window.addEventListener('resize', () => {
+      this.resizeTabs.bind(this)();
+      if (this.pinnedWidth > document.width / 2) {
```

Comment:
Hmmm…  This will shrink it, but not expand it if the person drags the window smaller then larger.
What about setting the property to the max of `this.pinnedWidth` and `document.width / 2` instead?